graph-manager-docs/source/integrations.md

```diff
@@ -119,83 +119,92 @@ The triggers you set up are evaluated on a rolling five minute window. For examp
 
 ## Datadog
 
-The Apollo Datadog integration allows you to forward the performance metrics that are available to you in Graph Manager to Datadog, which is convenient for teams already relying on Datadog for their monitoring. Datadog additionally supports an advanced function API, allowing teams to create sophisticated graphs and alerts for their GraphQL metrics.
+You can forward Graph Manager performance metrics to your Datadog account. Datadog supports an advanced function API, which enables you to create sophisticated graphs and alerts for GraphQL metrics.
 
-The Datadog metrics forwarded by Graph Manager are:
+### Setup
 
-- `apollo.engine.operations.count`: the number of GraphQL operations that were executed. This includes queries, mutations, and operations that resulted in an error.
-- `apollo.engine.operations.error_count`: the number of GraphQL operations that resulted in an error. This includes GraphQL execution errors, and HTTP errors if Graph Manager failed to connect to your server.
-- `apollo.engine.operations.cache_hit_count`: the number of GraphQL queries whose result was served from Apollo Graph Manager's full query cache.
-- A histogram of GraphQL operation response times, measured in milliseconds. Due to Graph Manager's aggregation method (logarithmic binning), these values are accurate to +/- 5%:
-  - `apollo.engine.operations.latency.min`
-  - `apollo.engine.operations.latency.median`
-  - `apollo.engine.operations.latency.95percentile`
-  - `apollo.engine.operations.latency.99percentile`
-  - `apollo.engine.operations.latency.max`
-  - `apollo.engine.operations.latency.avg`
+To integrate with Datadog, you provide your Datadog API key to Graph Manager. A  Datadog account with administrator privileges is required to obtain an API key.
 
-All metrics forwarded to Datadog are aggregated in 60-second intervals and tagged with the GraphQL operation name as `operation:<query-name>`. Unique query signatures with the same operation name are merged, and queries without an operation name are ignored.
+1. Go to your [Datadog Integrations page](https://app.datadoghq.com/account/settings) and select **Apollo Engine** from the list: 
 
-All of the metrics are also tagged with the Graph Manager graph ID as `service:<graph-id>` and the variant name as `variant:<variant-name>`, so multiple graphs from Graph Manager can send data to the same Datadog account. If you have not set a variant name, then "current" will be used.
+    ![DatadogIntegrationTile](./img/datadog/integration-tile.png)
 
-If you're reporting metrics to Graph Manager through the Engine proxy, Datadog will merge your statistics across multiple instances of the proxy (per-host metrics are not available). Just like in the Graph Manager UI, each operation inside a query batch is counted individually.
```

Comment:
This paragraph disappeared, was this intentional?

Reply:
It was! We've removed Engine Proxy docs elsewhere, and this continues that trend.